WcaOnRails/lib/country_city_validators/gb_city_validator.rb

```diff
@@ -0,0 +1,97 @@
+# frozen_string_literal: true
+
+GB_COUNTIES = %w(
+  Aberdeenshire
+  Angus
+  Argyll\ and\ Bute
+  Bedfordshire
+  Berkshire
+  Buckinghamshire
+  Cambridgeshire
+  Cheshire
+  City\ of\ Bristol
+  City\ of\ London
```

Comment:
Does this mean https://www.worldcubeassociation.org/competitions/LondonOpen2017 cityName should change from "London" to "London, City of London"?

Reply:
Would change to "Greater London". City of London is a tiny area right in the heart of London that is mostly office space and guild halls...

Reply:
Yeah, I would imagine in the UK, we would change the London 2017 city to something like "Docklands, Greater London"